Source/WTF/wtf/PlatformEnable.h

```diff
@@ -104,6 +104,10 @@
 
 /* Do not use PLATFORM() tests in this section ! */
 
+#if !defined(ENABLE_CONJECTURE_ASSERT)
+#define ENABLE_CONJECTURE_ASSERT 0
+#endif
```

Comment:
I don't think this is needed since `ENABLE` handles the value not being defined.

Reply:
I'll leave it just to document that this Enable option exists.